src/ontology/cl-edit.owl

```diff
@@ -28134,6 +28138,23 @@ SubClassOf(obo:CL_4023065 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
 SubClassOf(obo:CL_4023065 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
 SubClassOf(obo:CL_4023065 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P97367))
 
+
+# Class: obo:CL_4023069 (medial ganglionic eminence derived GABAergic cortical interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12637172") obo:IAO_0000115 obo:CL_4023069 "A GABAergic cortical interneuron that develops from the medial ganglionic eminence and has migrated to the cerebral cortex.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023069 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023069 "MGE cortical interneuron")
+AnnotationAssertion(rdfs:label obo:CL_4023069 "medial ganglionic eminence derived GABAergic cortical interneuron")
+EquivalentClasses(obo:CL_4023069 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002202 obo:UBERON_0004024) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534)))
+
```

Comment:
develops_from some 'medial ganglionic eminence' should be a clause in the EquivalentClass expression.  

As currently written you'll get inference that *ALL* GABAergic cortical interneurons develop_from the MGE!

This pattern -  mixing EC and SC axioms -  is powerful and should be used sparingly as it can be hard to track where inferences are coming from when it is used. 

(The pattern is known as a hidden GCI (General Class Inclusion axiom). In GCI form it looks like this:

'develops from' some 'medial ganglionic eminence'  subClassOf:  'GABAergic Interneuron' and 'has soma location' some cortex.

Using a GCI would allow you to state this without making a named class equivalentTo the class expression on the right.)

Reply:
Just looked again.  Your axioms are fine.  I misread the functional syntax (fooled by using the side-by-side diff view with softwrap into thinking these were two separate lines).

Reply:
would still like to learn about GCI though, will ask you about it later at the meeting perhaps